src/docs/issuing-assets/how-to-issue-an-asset.mdx

```diff
@@ -0,0 +1,193 @@
+---
+title: How to issue an asset
+order: 20
+---
+
+## High-level view of the steps
+There is no dedicated operation to create an asset on Stellar.  Instead, assets are created with a payment operation: an issuing account makes a payment using the asset it’s issuing, and that payment actually creates the asset on the network.
+
+It’s a pretty simple process that requires four operations: one to create an issuing account, one to create a distribution account, one to establish a trustline, and one to make a payment. 
+
+The code to create those operations and submit them as transactions is below.  Here, we’ll walk through each step so that the process makes sense.  You can breeze through to get a general understanding, or you can use the [Stellar laboratory](https://www.stellar.org/laboratory/), which is an interface that allows you to create and submit transactions, to actually follow along and issue a token right here right now.
+
+One caveat: if you are creating a token on the public network, there is one additional prerequisite.  You need a funded account to provide the XLM necessary to create the issuing and distribution accounts.
+
+### Create the issuing and distribution accounts
```

Comment:
if you want these H3 titles to show up in the right sidenav page table of contents, they'll need to be H2s.

Reply:
Dang.  This page has 2 main sections, each representing a different way to view the same process: from the high level, and from the code level.  These h3s are subsections of `High-level view`, and moving them up to h2 obscures that fact. 

A lot of the docs rely on subsections for clarity.  Any way to add indented h3s to the right sidenav?

Reply:
sure! @vcarl @jeesunikim thoughts on adding H3s to the right sidenav on regular docs?

Reply:
Sure, sounds good to me!